### PR TITLE
Promote: String standards

### PR DESCRIPTION
**Pull request by panpawn**
And also add a missing quote mark for one of the "Did you mean "/globalrank" type replies.

### CODE REVIEW

File: chat-commands.js [hunk 1]
```diff
@@ -1866,39 +1866,39 @@ exports.commands = {
...
 		Users.setOfflineGroup(name, nextGroup);
 		if (Config.groups[nextGroup].rank < Config.groups[currentGroup].rank) {
-			this.privateModCommand("(" + name + " was demoted to " + groupName + " by " + user.name + ".)");
-			if (targetUser) targetUser.popup("You were demoted to " + groupName + " by " + user.name + ".");
+			this.privateModCommand(`(${name} was demoted to ${groupName} by ${user.name}.)`);
+			if (targetUser) targetUser.popup(`You were demoted to ${groupName} by ${user.name}.`);
 		} else {
-			this.addModCommand("" + name + " was promoted to " + groupName + " by " + user.name + ".");
-			if (targetUser) targetUser.popup("You were promoted to " + groupName + " by " + user.name + ".");
+			this.addModCommand(`${'' + name} was promoted to ${groupName} by ${user.name}.`);
```
Comment (Zarel) [inline chat-commands.js]: `'' + name`?

Comment (panpawn) [inline chat-commands.js]: how should I concatenate there?

Comment (panpawn) [inline chat-commands.js]: https://github.com/Zarel/Pokemon-Showdown/blob/master/chat-commands.js#L1132

Comment (Zarel) [inline chat-commands.js]: That's nearly definitely a mistake. It should just be `${name}`

**Comment by Zarel** [comment]
@panpawn, you let this lapse enough to get a merge conflict... :( Can you fix this or should I close it?

**Comment by panpawn** [comment]
ok

**Pull request merged**